apps/web/app/[lang]/sponsors/components/About.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, NativeH1, P1, P2 } from 'ui/core';
+import SemiCircleLeft from '@public/images/shared/semi-circle-left.png';
+import SemiCircleGray from '@public/images/shared/semi-circle-gray.png';
+import { ImageWithFallback } from '@components/utils/ImageWithFallback';
+import { useTranslations } from 'intl';
+
+export const About = () => {
+  const t = useTranslations('sponsors.about');
+
+  return (
+    <div className="w-full flex flex-col gap-[72px] pb-16">
+      <div className="w-full flex justify-between items-stretch p-4">
+        <div className="flex flex-col justify-around md:w-1/2 gap-8 py-6">
```

Comment:
```suggestion
        <div className="flex flex-col justify-around md:w-1/2 gap-8 md:py-6">
```

Remove that padding unless we are on "md" or the correct breakpoint in which you show the shapes